spec/features/requester_edits_ncr_work_order_spec.rb

```diff
@@ -7,10 +7,11 @@
 
   let(:work_order) { create(:ncr_work_order, description: 'test') }
   let(:ncr_proposal) { work_order.proposal }
-  let!(:approver) { create(:user) }
+  let!(:approver) { create(:user, client_slug: 'ncr') }
```

Comment:
I dunno. Doesn't save much typing. It's not quite a constant use case. The string itself is user data (stored as a string in the db) and so doing comparisons might require a cast? Validation assumes string comparison? Maybe? Not obvious (to me) that it's a clear win.